modules/fastq/templates/merge_cram.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -euo pipefail
+
+main() {
+    IFS=' ' read -a cram_array <<< "!{crams}";
+    if [ ${#cram_array} -gt 1 ]
+    then
+        ${CMD_SAMTOOLS} merge -@ "!{task.cpus}" -o !{cramOut} --write-index "${cram_array}"
+    else
+        mv !{crams} "!{cramOut}"
```

Comment:
I think that this will move the Nextflow input (a file in another process in the workdir) leading to unexpected results and causing issues with resuming the pipeline.

Reply:
I think so too, I'm sure I fixed it before, I'll take a look